benches/bench_flows.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from pytest_benchmark.fixture import BenchmarkFixture
+
+from prefect import flow, task
+
+
+def noop_function():
+    pass
+
+
+def bench_flow_decorator(benchmark: BenchmarkFixture):
+    benchmark(flow, noop_function)
+
+
+@pytest.mark.parametrize("options", [{}, {"timeout_seconds": 10}])
+def bench_flow_call(benchmark: BenchmarkFixture, options):
+    noop_flow = flow(**options)(noop_function)
+    benchmark(noop_flow)
+
+
+@pytest.mark.parametrize("num_tasks", [10, 50, 100, 200])
+def bench_flow_with_submitted_tasks(benchmark: BenchmarkFixture, num_tasks: int):
+    test_task = task(noop_function)
+
+    @flow
+    def test_flow():
+        for _ in range(num_tasks):
+            test_task.submit()
+
+    benchmark(test_flow)
+
+
+@pytest.mark.skip(reason="Runs longer than the 60s timeout")
```

Comment:
That we can't run 200 no-op tasks in 60s seems like a great, but sad, place to start benchmarking.

Reply:
It's running multiple iterations of the benchmark to get more stable numbers, so the "test" exceeds that timeout but each bench itself does not. I'm going to bump the timeout here.